noxfile.py

```diff
@@ -225,3 +248,29 @@ def itde_unit_tests(session: nox.Session) -> None:
 
     context = _context(session, coverage=True)
     _unit_tests(session, PROJECT_CONFIG, context)
+
+
+@nox.session(name="update-default-db-version", python=False)
+def update_default_db_version(session: nox.Session):
+    is_ok = True
+    p = ArgumentParser(
+        usage='nox -s update-default-db-version -- --version "major.minor.patch"',
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+    )
+    p.add_argument("--version")
+    args = p.parse_args(session.posargs)
+    new_version = args.version
+
+    # Get default db version from python file
+    pwd = os.getcwd()
+    file_name = os.path.join(
+        pwd,
+        "exasol_integration_test_docker_environment/cli/options/test_environment_options.py",
+    )
+    default_db_ver = get_default_db_version(file_name)
+    is_ok = is_ok and default_db_ver
+    is_ok = is_ok and replace_string_in_file(file_name, default_db_ver, new_version)
```

Comment:
I'am thinking if it would be better to replace the whole string:
```
LATEST_DB_VERSION = """<old_version"""
```
with 
```
LATEST_DB_VERSION = """<new_version"""
```

in order to avoid any unwanted changes.